### PR TITLE
Added heat conversion functionality

### PR DESCRIPTION
**Pull request by AndrewMollett**
Please review

### CODE REVIEW

**Comment by AndrewMollett** [comment]
Seperated the heat conversion and base code into seperate files now.

File: heat-converter.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+'use strict';
+//Convert celcius to farenheight
+var farenConverter = function(number){
```
Comment (keithbarrow) [inline heat-converter.js]: full names please - "faren" isn't explicit. Real teams will be this picky.

File: heat-converter.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+    number *= 1.8;
+    number += 32;
+    return number;
+};
+
+var celcConverter = function(number){
```
Comment (keithbarrow) [inline heat-converter.js]: again  - celcius, not celc

**Comment by keithbarrow** [comment]
Two comments - please fix and feel free to :shipit: after

**Pull request merged**